typegraph/node/dev/fix-declarations.ts

```diff
@@ -6,22 +6,15 @@ import {
   fromFileUrl,
   resolve,
 } from "https://deno.land/std@0.202.0/path/mod.ts";
-import { expandGlobSync } from "https://deno.land/std@0.202.0/fs/mod.ts";
 
 export const thisDir = dirname(fromFileUrl(import.meta.url));
```

Comment:
with jco latest (upgrade), you can remove this entirely

Reply:
The fix-declarations.ts script?

Reply:
yes